client/public/index.html

```diff
@@ -9,6 +9,8 @@
       content="Web site created using create-react-app"
     />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <script src="https://kit.fontawesome.com/90a9855ce2.js" crossorigin="anonymous"></script>
```

Comment:
I have seen this tends to increase the rendering time. Correct me if I'm wrong.
What can be done here?
Add static svg of these icons instead

Reply:
Yes I agree, adding static svg would decrease the rendering time , working on it !! . Thanks for the suggestion

Reply:
Issue resolved : static svg icons added .